pkg/asset/installconfig/vsphere/client.go

```diff
@@ -0,0 +1,24 @@
+package vsphere
+
+import (
+	"context"
+
+	"github.com/vmware/govmomi/find"
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/govmomi/vim25"
+)
+
+type VSphereClient interface {
```

Comment:
We are using the term "finder" for this everywhere else. Let's stay consistent.
```suggestion
type Finder interface {
```